src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -38,17 +38,13 @@ class ExpressionValidator extends ConstraintValidator
     private $expressionLanguage;
 
     /**
-     * @param PropertyAccessorInterface|null $propertyAccessor Optional as of Symfony 2.5
-     *
-     * @throws UnexpectedTypeException If the property accessor is invalid
+     * @param PropertyAccessorInterface|null $propertyAccessor
```

Comment:
Then I suggest removing the doc block, it provide no value, pure repetition of the signature